internal/model/modeldecoder/nullable/nullable.go

```diff
@@ -27,6 +27,12 @@ import (
 	jsoniter "github.com/json-iterator/go"
 )
 
+var supportedTSFormats = []string{
```

Comment:
Could you please add a comment explaining that the below are variations of time.RFC3339, supporting more formats for the timezone offset?